### PR TITLE
chore: create issue fingerprint table in postgres

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

Related to Related to https://github.com/PostHog/product-internal/pull/662

## Changes

Adds a new Postgres table that tracks any issue fingerprint overrides. This is the source of truth from which the ClickHouse table is based.

### CODE REVIEW

File: posthog/models/error_tracking/error_tracking.py [hunk 1]
```diff
@@ -58,3 +58,15 @@ def convert_fingerprints_to_tuples(fps: list[list[str]]):
         # converting back to list of lists before saving
         self.merged_fingerprints = [list(f) for f in merged_fingerprints]
         self.save()
+
+
+class ErrorTrackingIssueFingerprint(models.Model):
+    team = models.ForeignKey("Team", on_delete=models.CASCADE, db_index=False)
+    issue = models.ForeignKey(ErrorTrackingGroup, on_delete=models.CASCADE)
+    fingerprint = models.CharField(max_length=400)
```
Comment (daibhin) [inline posthog/models/error_tracking/error_tracking.py:66]: Going with a string here. I know fingerprints are arrays of strings right now but I feel like the intention is to move towards a proper hash that will be stored as a string

File: posthog/models/error_tracking/error_tracking.py [hunk 2]
```diff
@@ -58,3 +58,15 @@ def convert_fingerprints_to_tuples(fps: list[list[str]]):
...
+
+    # current version of the id, used to sync with ClickHouse and collapse rows correctly for overrides ClickHouse table
+    version = models.BigIntegerField(null=True, blank=True)
```
Comment (tkaemming) [inline posthog/models/error_tracking/error_tracking.py:69]: This probably shouldn't be nullable (we always want these records to be versioned.)

Comment (daibhin) [reply posthog/models/error_tracking/error_tracking.py:69]: Note to self: don't [copy persons](https://github.com/PostHog/posthog/blob/f31abb5787f7c516bafcd515abe65b68b86a3cba/posthog/models/person/person.py#L126) so directly 😂

File: posthog/models/error_tracking/error_tracking.py (hunk 1 above)
Comment (oliverb123) [reply posthog/models/error_tracking/error_tracking.py:66]: 400 is a "magic number" dotted around our codebase whose significance I don't understand.

**Review by oliverb123 — APPROVED** [review]

**Pull request merged**